### PR TITLE
new isonline util function

### PR DESCRIPTION
**Pull request by garrensmith**
THis is just a proof of concept for now.

### CODE REVIEW

**Comment by robertkowalski** [comment]
looks good, cool idea

File: test/helpers.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
+import nock from 'nock';
+
+export function nodeIsOnline (url) {
```
Comment (robertkowalski) [inline test/helpers.js]: can we rename it to `fakeNodeIsOnline`, `mockNodeIsOnline` or something along the lines?

**Comment by robertkowalski** [comment]
+1 after renaming the file

**Comment by robertkowalski** [comment]
oh!

`isOnline` is not used for the query command

**Comment by garrensmith** [comment]
Merged

**Pull request closed without merging**